README.md

```diff
@@ -36,6 +36,15 @@ For more information about features go to [our Swift documentation](https://meil
 
 ## Get started
 
+### Cocoapods
+
+**MeiliSearch-Swift** is available through [CocoaPods](https://cocoapods.org). To install
+it, simply add the following line to your Podfile:
+
+```ruby
+pod 'MeiliSearch'
+```
```

Comment:
We don't need to add a version?

Reply:
This is not required. If the version is not explicit it always get the latest release.